unlock-app/package.json

```diff
@@ -81,8 +81,8 @@
     "deploy": "next export src -o out",
     "deploy-netlify": "./scripts/deploy-netlify.sh",
     "deploy-unlock-contract": "node scripts/deploy-unlock.js",
-    "start": "cross-env NODE_ENV=production node src/server.js",
```

Comment:
Is this an intentional change? IIRC we have that so the project builds on Windows

Reply:
Damit. That was not meant to be committed. Thanks for the sharp eye!